stl/inc/array

```diff
@@ -778,11 +778,21 @@ _NODISCARD _CONSTEXPR20 bool operator==(const array<_Ty, _Size>& _Left, const ar
     return _STD equal(_Left.begin(), _Left.end(), _Right.begin());
 }
 
+#if !_HAS_CXX20
 template <class _Ty, size_t _Size>
 _NODISCARD _CONSTEXPR20 bool operator!=(const array<_Ty, _Size>& _Left, const array<_Ty, _Size>& _Right) {
     return !(_Left == _Right);
 }
+#endif // !_HAS_CXX20
 
+#ifdef __cpp_lib_concepts
+template <class _Ty, size_t _Size>
+_NODISCARD constexpr _Synth_three_way_result<_Ty> operator<=>(
+    const array<_Ty, _Size>& _Left, const array<_Ty, _Size>& _Right) {
+    return _STD lexicographical_compare_three_way(
+        _Left.begin(), _Left.end(), _Right.begin(), _Right.end(), _Synth_three_way{});

```

Comment:
Should this call use `_Unchecked_begin`/`_Unchecked_end`? (And on 778 and 798?)

Reply:
Thanks! We hadn't updated the comparisons after adding the unchecked functions. I'll fix them all up.